### PR TITLE
DEVDOCS-5012: [update] Webhook Events, add metafield webhooks

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-5012]
{Ticket number or summary of work}

## What changed?
Provide a bulleted list in the present tense
* 

## Anything else?
Add related PRs, salient notes, ticket numbers, etc.

ping {names}

[DEVDOCS-5012]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5012?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/api-docs/webhooks/webhook-events.mdx [hunk 1]
```diff
@@ -303,27 +413,27 @@ Consult the [notifications section of the Channel Webhooks Guide](/api-docs/chan
...
-| store/order/* | Subscribe to all store/order events.  |
+| store/order/* | Fires when subscribed to all store/order events.  |
 | store/order/created |Fires if an order is created using the control panel, an app, or the API.|
 | store/order/updated| Fires when an already created order is updated. Any changes to an existing order will fire this webhook. Updates can include changing the status, updating a coupon, or changing an address.|
-| store/order/archived | Order is archived. |
+| store/order/archived | Fires when an order is archived. |
 | store/order/statusUpdated | This will only fire if the order status has changed, such as Pending to Awaiting Payment. |
-| store/order/message/created | Order message is created by customer or in control panel. |
-| store/order/refund/created | A refund has been submitted against an order. |
+| store/order/message/created | Fires if order message is created by a customer or in control panel. |
```
Comment (bc-tgomez) [inline docs/api-docs/webhooks/webhook-events.mdx:421]:
```suggestion
| store/order/message/created | Fires if order message is created by a customer or in the control panel. |
```

**Review by bc-tgomez — APPROVED** [review]

File: docs/api-docs/webhooks/webhook-events.mdx [hunk 2]
```diff
@@ -194,13 +273,13 @@ Consult the [channel events section of the Channel Webhooks Guide](/api-docs/cha
...
 | Name / Scope             | Description |
 |:-------------------------|:------------|
-| store/customer/* | Subscribe to all store/customer events. |
-| store/customer/created | A new customer is created.|
-| store/customer/updated | Customer is updated. Does not currently track changes to the customer address. |
-| store/customer/deleted | Customer is deleted. |
-| store/customer/address/created | Customer address is created. |
-| store/customer/address/updated | Customer address is updated. |
-| store/customer/address/deleted | Customer address is deleted. |
+| store/customer/* | This webhook fires when subscribed to all store/customer events. |
```
Comment (slsriehl) [inline docs/api-docs/webhooks/webhook-events.mdx:276]:
```suggestion
| store/customer/* | Fires for all store/customer events. |
```

File: docs/api-docs/webhooks/webhook-events.mdx [hunk 3]
```diff
@@ -254,6 +333,36 @@ Payload objects with the following scopes take the form that follows:
 
 Consult the [emails section of the Channel Webhooks Guide](/api-docs/channels/guide/webhooks#emails).
 
+## Inventories
+
+| Name / Scope             | Description |
+|:-------------------------|:------------|
+| store/inventory/location/metafield/*        | This webhook fires when subscribed to all store/inventory/location/metafields events.  |
```
Comment (slsriehl) [inline docs/api-docs/webhooks/webhook-events.mdx:340]:
```suggestion
| store/inventory/location/metafield/*        |  Fires for all store/inventory/location/metafields events.  |
```

File: docs/api-docs/webhooks/webhook-events.mdx [hunk 4]
```diff
@@ -754,18 +866,18 @@ Changes to the following store settings will trigger a `store/information/update
 
 | Name / Scope             | Description |
 |:-------------------------|:------------|
-| store/subscriber/*       | Subscribe to all store/subscriber events. |
-| store/subscriber/created | Subscriber is created. |
-| store/subscriber/updated | Subscriber is updated. |
-| store/subscriber/deleted | Subscriber is deleted. |
+| store/subscriber/*       | This webhook fires when subscribed to all store/subscriber events. |
```
Comment (slsriehl) [inline docs/api-docs/webhooks/webhook-events.mdx:869]:
```suggestion
| store/subscriber/*       | Fires for all store/subscriber events. |
```

File: docs/api-docs/webhooks/webhook-events.mdx [hunk 5]
```diff
@@ -33,6 +33,35 @@ Callback payloads take the following form. The `data` property will contain vary
   "producer": "stores/{store_hash}"
 }
 ```
+## Brands
+
+| Name / Scope            | Description |
+|:------------------------|:------------|
+| store/brand/metafield/* | This webhook will fire when subscribed to all brand events. |
```
Comment (slsriehl) [inline docs/api-docs/webhooks/webhook-events.mdx:40]:
```suggestion
| store/brand/metafield/* | Fires for all store/brand/metafield events. |
```

File: docs/api-docs/webhooks/webhook-events.mdx [hunk 6]
```diff
@@ -154,10 +206,15 @@ Payload objects with the following scopes take the form that follows:
 
 | Name / Scope            | Description |
 |:------------------------|:------------|
-| store/category/* | Subscribe to all store/category events. |
-| store/category/created | Category is created. |
-| store/category/updated | Category is updated. |
-| store/category/deleted | Category is deleted. |
+| store/category/* | This webhook fires when subscribed to all store/category events. |
```
Comment (slsriehl) [inline docs/api-docs/webhooks/webhook-events.mdx:209]:
```suggestion
| store/category/* | Fires for all store/category events. |
```

File: docs/api-docs/webhooks/webhook-events.mdx [hunk 7]
```diff
@@ -262,22 +371,23 @@ Consult the [inventory section of the _Buy Online, Pick up in Store_ Webhook Eve
 
 Consult the [locations section of the _Buy Online, Pick up in Store_ Webhook Events](/buy-online-pick-up-in-store/webhooks#locations).
 
+
 ## Metafields
 
 | Name / Scope             | Description |
 |:-------------------------|:------------|
-| store/metafield/*        | Subscribe to all store/metafields events.  |
+| store/metafield/*        | This webhook fires when subscribed to all store/metafields events.  |
```
Comment (slsriehl) [inline docs/api-docs/webhooks/webhook-events.mdx:379]:
```suggestion
| store/metafield/*        | Fires for all store/metafields events.  |
```

File: docs/api-docs/webhooks/webhook-events.mdx [hunk 8]
```diff
@@ -303,27 +413,27 @@ Consult the [notifications section of the Channel Webhooks Guide](/api-docs/chan
 
 | Name / Scope             | Description |
 |:-------------------------|:------------|
-| store/order/* | Subscribe to all store/order events.  |
+| store/order/* | Fires when subscribed to all store/order events.  |
```
Comment (slsriehl) [inline docs/api-docs/webhooks/webhook-events.mdx:416]:
```suggestion
| store/order/* | Fires for all store/order events.  |
```

File: docs/api-docs/webhooks/webhook-events.mdx [hunk 9]
```diff
@@ -303,27 +413,27 @@ Consult the [notifications section of the Channel Webhooks Guide](/api-docs/chan
...
+| store/order/archived | Fires when an order is archived. |
 | store/order/statusUpdated | This will only fire if the order status has changed, such as Pending to Awaiting Payment. |
-| store/order/message/created | Order message is created by customer or in control panel. |
-| store/order/refund/created | A refund has been submitted against an order. |
+| store/order/message/created | Fires if order message is created by a customer or in the control panel. |
+| store/order/refund/created | Fires if a refund has been submitted against an order. |
 | store/order/transaction/created | Fires when a transaction record is created. |
 | store/order/transaction/updated | Fires when a transaction record is updated.  |
-| store/order/metafield* | Subscribe to all store/order/metafield events.  |
+| store/order/metafield* | Fires when subscribed to all store/order/metafield events.  |
```
Comment (slsriehl) [inline docs/api-docs/webhooks/webhook-events.mdx:425]:
```suggestion
| store/order/metafield* | Fires for all store/order/metafield events.  |
```

File: docs/api-docs/webhooks/webhook-events.mdx [hunk 10]
```diff
@@ -451,20 +561,20 @@ Consult the [product assignment section of the Channel Webhooks Guide](/api-docs
 
 | Name / Scope             | Description |
 |:-------------------------|:------------|
-| store/product/* | Subscribe to all store/product events. |
-| store/product/deleted| Product is deleted.|
-| store/product/created | A new product is created. |
-| store/product/updated |Occurs when product details are edited. For a full list of product fields that trigger an updated event, consult the following section on [product updated events](#product-update-events). |
-| store/product/inventory/updated| Product inventory is updated. |
+| store/product/* | Fires when subscribed to all store/product events. |
```
Comment (slsriehl) [inline docs/api-docs/webhooks/webhook-events.mdx:564]:
```suggestion
| store/product/* | Fires for all store/product events. |
```

File: docs/api-docs/webhooks/webhook-events.mdx [hunk 11]
```diff
@@ -451,20 +561,20 @@ Consult the [product assignment section of the Channel Webhooks Guide](/api-docs
...
+| store/product/deleted| Fires when a product is deleted.|
+| store/product/created | Fires when a new product is created. |
+| store/product/updated | Occurs when product details are edited. For a full list of product fields that trigger an updated event, consult the following section on [product updated events](#product-update-events). |
+| store/product/inventory/updated| Fires when product inventory is updated. |
 | store/product/inventory/order/updated| Fires if a product’s inventory is decremented or incremented, including when an order is placed. Webhook responds to inventory updates made using the control panel, CSV import, API or an app.|
-| store/product/metafield/* | Subscribe to all store/product/metafield events. |
-| store/product/metafield/created | A new product metafield is created. |
+| store/product/metafield/* | Fires when subscribed to all store/product/metafield events. |
```
Comment (slsriehl) [inline docs/api-docs/webhooks/webhook-events.mdx:570]:
```suggestion
| store/product/metafield/* | Fires for all store/product/metafield events. |
```

File: docs/api-docs/webhooks/webhook-events.mdx [hunk 12]
```diff
@@ -451,20 +561,20 @@ Consult the [product assignment section of the Channel Webhooks Guide](/api-docs
...
+| store/product/metafield/created | Fires when a new product metafield is created. |
 | store/product/metafield/updated | Occurs when product metafield details are edited. |
-| store/product/metafield/deleted | A product metafield is deleted.|
-| store/variant/metafield/* | Subscribe to all store/variant/metafield events. |
-| store/variant/metafield/created | A new product variant metafield is created. |
+| store/product/metafield/deleted | Fires when a product metafield is deleted.|
+| store/variant/metafield/* | Fires when subscribed to all store/variant/metafield events. |
```
Comment (slsriehl) [inline docs/api-docs/webhooks/webhook-events.mdx:574]:
```suggestion
| store/variant/metafield/* | Fires for all store/variant/metafield events. |
```

File: docs/api-docs/webhooks/webhook-events.mdx [hunk 13]
```diff
@@ -626,15 +737,16 @@ Payload objects with the following scopes take the form that follows:
...
 ## SKUs
 
 | Name / Scope             | Description |
 |:-------------------------|:------------|
-| store/sku/* | Subscribe to all store/sku events. |
-| store/sku/created | A new sku is created. |
-| store/sku/updated | SKU is updated. |
-| store/sku/deleted | SKU is deleted. |
-| store/sku/inventory/updated | SKU is updated.|
+| store/sku/* | This webhook fires when subscribed to all store/sku events. |
```
Comment (slsriehl) [inline docs/api-docs/webhooks/webhook-events.mdx:745]:
```suggestion
| store/sku/* | Fires for all store/sku events. |
```

**Comment by slsriehl** [comment]
closing for git issues. Please open again, @bc-traciporter

**Pull request closed without merging**